Sources/firebase/include/FirebaseCore.hh

```diff
@@ -35,6 +35,21 @@ class SWIFT_CONFORMS_TO_PROTOCOL(FirebaseCore.FutureProtocol)
           completion(user_data);
         });
   }
+
+  // FIXME: Remove once https://github.com/apple/swift/issues/74578 is fixed.
+  int error() const {
```

Comment:
Can we block the region with `#if defined(SR74578)` and add `-D SR74578` and/or `-Xcc -DSR74758` so that it is more obvious that this is a workaround?

Reply:
Good idea!